test/avocado/selenium-sosreport.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
+
+# we need to be able to find and import seleniumlib, so add this directory
+import os
+import sys
+import time
+machine_test_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(1, machine_test_dir)
+
+from avocado import main
+from avocado.utils import process
+import libdisc
+from seleniumlib import *
+
+class SosReportingTab(SeleniumTest):
+    """
+    :avocado: enable
+    """
+    def test10SosReport(self):
+        self.login()
+        self.wait_id("sidebar")
+        self.wait_id("tools-panel",cond=invisible)
+        self.click(self.wait_link('Tools', cond=clickable))
+        self.wait_id("tools-panel")
+        self.click(self.wait_link('Diagnostic report', cond=clickable))
+        self.wait_frame("sosreport")
+        self.wait_text("This tool will collect system configuration and diagnostic")
+        self.click(self.wait_xpath('//button[@data-target="#sos"]', cond=clickable))
+        self.wait_id("sos")
+        self.wait_text("Generating report")
+        process.run("pgrep sosreport", shell=True)
+        while True:
```

Comment:
I can add comment, but it seems that it is easy to understand why there is that part of code :-)
Otherwise, I have there only one process.run, without these things along, but on verifymachines it caused that I was disconnected from selenium hub, becuase this active waiting was longer than timeout for selenium grid hub. Thats why I also call some selenium function, to do not lost connection (I can also comment that)